### DISCUSSION TITLE
Certain windows.exe files not running on MacOS Ventura using the latest version of Wineskin

### DISCUSSION THREAD

**Discussion by Averlief** [The-Wineskin-Project/WineskinServer#448] [Q&A]
Hello, I have an M2 MacBook Air that runs MacOS Ventura. I downloaded the latest version Wineskin (2.9.1.8) to create application bundles to run 32 bit applications as WineBottler will not work on MacOS Ventura. While I did get some 32 bit .exe Windows files to run, some other 32 bit .exe files simply would not run and are seemingly not readable via Wineskin or all other equivalents of Wineskin that are compatible with MacOS Ventura. Why is this? Are there any Winetricks or commands that need to be executed while creating the application bundle? If so, what are they? I'd appreciate it if anyone could help me out with this. Thanks.

**Comment by emendelson** [comment]
On my M2 MacBook Air running Ventura, I used Wineskin Winery to create an executable and installed the 32-bit Windows app vDos.exe. it worked perfectly.

Have you restarted your machine? Sometimes, when something goes wrong with wine, nothing will work until after a restart.

**Reply by Averlief** [reply]
Thanks for your response, emendelson. I did restart my machine, but for certain 32 bit .exe files, I get the message 'no executables found' while for other 32 bit .exe files, they run perfectlylike vDos.exe did in your case.

What version of Wineskin do you use? Did you need to install any Winetricks or execute any commands?

**Comment by emendelson** [comment]
No Winetricks; no commands. I simply created a blank wrapper and let Wineskin run the setup executable.

Wineskin Winery 1.8.4.2
Wrapper Wineskin 2.9.1.9
Engine WS11WinCX64Bit22.1.1-10

Also, all my existing wrappers run correctly.

**Reply by Averlief** [reply]
If it's at all helpful, I tried running those same files using the same wrapper on MacOS Mojave and I encountered the same result, i.e those particular .exe files would not run. Interestingly, an older wrapper ran those .exe files on MacOS Mojave but that wrapper doesn't appear to be compatible with MacOS Catalina and newer operating systems. On MacOS Ventura, when I tried to open the older wrapper, I would get the message 'Wineskin is currently busy' with a spinning circle. 

Do you happen to know if there's a way to get the older wrapper to run on MacOS Ventura and if Wineskin will be providing any updates that address this issue of certain .exe files not running on the newer wrappers?
